linchpin/cli/__init__.py

```diff
@@ -374,7 +374,8 @@ def lp_down(self, pinfile, targets=(), run_id=None):
         pass
 
 
-    def lp_up(self, targets=(), run_id=None, tx_id=None, inv_f="cfg"):
+    def lp_up(self, targets=(), run_id=None, tx_id=None, inv_f="cfg",
+              vault_pass=''):
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.